bcss_notify_callback/lambda_function.py

```diff
@@ -0,0 +1,273 @@
+import boto3
+import json
+import hashlib
+import hmac
+import logging
+import os
+import requests
+import bcss_notify_callback.sql as sql
+import oracle_connection as db
+from bcss_notify_callback.patients_to_update import patient_to_update
+from typing import Dict, Any
+
+REGION_NAME = os.getenv("region_name")
+logging.basicConfig(
+    format="{asctime} - {levelname} - {message}", style="{", datefmt="%Y-%m-%d %H:%M:%S"
+)
+logger = logging.getLogger()
```

Comment:
Very minor but we're using the default logger here so these could just be `logging.error`, `logging.info` and `logging.debug` statements without initialising the default logger.